src/bandersnatch/mirror.py

```diff
@@ -112,17 +113,27 @@ def webdir(self) -> Path:
     def todolist(self) -> Path:
         return self.homedir / "todo"
 
-    async def synchronize(self) -> Dict[str, Set[str]]:
+    async def synchronize(
+        self, specific_packages: Optional[List[str]] = None
+    ) -> Dict[str, Set[str]]:
         logger.info(f"Syncing with {self.master.url}.")
         self.now = datetime.datetime.utcnow()
         # Lets ensure we get a new dict each run
         # - others importing may not reset this like our main.py
         self.altered_packages = {}
 
-        await self.determine_packages_to_sync()
-        await self.sync_packages()
-        self.sync_index_page()
-        self.wrapup_successful_sync()
+        if specific_packages is None:
+            # Changelog-based synchronization
+            await self.determine_packages_to_sync()
+            await self.sync_packages()
+            self.sync_index_page()
+            self.wrapup_successful_sync()
+        else:
+            self.packages_to_sync = {
+                utils.bandersnatch_safe_name(name): 1 for name in specific_packages
```

Comment:
What does 1 mean here? Should we make a variable to help explain that? 
```suggestion
                utils.bandersnatch_safe_name(name): LOW_SERIAL for name in specific_packages
```